File: Python/fibonacci.py
def fibonacci(n):
    # Initialize the first two terms of the sequence
    fib_sequence = [0, 1]
    # Check if n is less than or equal to 0
    if n <= 0:
        return []
    # Check if n is 1, return the first term
    elif n == 1:
        return [0]
    # Generate the Fibonacci sequence up to the nth term
    while len(fib_sequence) < n:
        next_term = fib_sequence[-1] + fib_sequence[-2]
        fib_sequence.append(next_term)

    return fib_sequence

# Example: Print the first 10 Fibonacci numbers
n = 10
fib_series = fibonacci(n)
print(fib_series)

File: Rust/Fibonacci.rs
fn fibonacci(n: u64) -> u64 {
    if n == 0 {
        return 0;
    } else if n == 1 {
        return 1;
    } else {
        return fibonacci(n - 1) + fibonacci(n - 2);
    }
}

fn main() {
    let n = 10; // Change this to the desired Fibonacci sequence length
    for i in 0..n {
        println!("Fibonacci({}) = {}", i, fibonacci(i));
    }
}